shared/chat/conversation/list.desktop.js

```diff
@@ -410,6 +410,41 @@ class ConversationList extends Component<void, Props, State> {
     }
   }
 
+  _domNodeToRect (element) {
+    const bodyRect = document.body.getBoundingClientRect()
+    const elemRect = element.getBoundingClientRect()
+
+    return {
+      height: elemRect.height,
+      left: elemRect.left - bodyRect.left,
+      top: elemRect.top - bodyRect.top,
+      width: elemRect.width,
+    }
+  }
+
+  onEditLastMessage = () => {
+    if (!this._list) {
+      return
+    }
+
+    const entry: any = this.state.messages.findLastEntry(m => m.type === 'Text' && m.author === this.props.you)
```

Comment:
i had to use any here for flow not to freak out, unclear to me why its this confused. @MarcoPolo ?